yubico-piv-tool.json

```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://developers.yubico.com/yubico-piv-tool/",
+    "version": "1.5.0",
+    "license": "BSD 2-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-1.5.0-win64.zip",
+            "hash": "d51fdc7dc1f717e1b0fafb503e999377b95496c4fa2d1686e37fbbb063f9ca15"
+        },
+        "32bit": {
+            "url": "https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-1.5.0-win32.zip",
+            "hash": "74695de467b13d7320b99ac85a04ff24ae18c7d3230f71aa85fda07feba426d2"
+        }
+    },
+    "extract_dir": "",
+    "bin": [
+        "bin\\yubico-piv-tool.exe"
```

Comment:
Since they’re both CLI, do these two apps belong in the main bucket instead?

Reply:
Oh, CLI tool shoud belong to main bucket ? I didn't know that.

Reply:
Yes, can you submit this PR to 
https://github.com/lukesampson/scoop instead?